### ISSUE TITLE
Question about the training time in log file

### ISSUE THREAD

**Issue by LZDSJTU** [tianweiy/CenterPoint#293]
In the training log:
INFO - Epoch [1/3][30/4934]   lr: 0.00030, eta: 9:39:15, time: 0.715, data_time: 0.047, transfer_time: 0.035, forward_time: 0.388, loss_parse_time: 0.000 memory: 7375.

However, the time (0.715) is not equal to data_time (0.047) + transfer_time (0.035) + forward_time (0.388) + loss_parse_time (0.000). It seems that the extra time is caused by optimizer. But when I comment the optimizer.step(), the time has no change.

I am not sure what causes the problem.

**Comment by tianweiy** [comment]
loss.backward takes the remained time
